packages/ui-core/common/src/lib/http-params/build-http-params.ts

```
@@ -0,0 +1,48 @@
+import { HttpParams } from '@angular/common/http';
+import { CustomHttpParamEncoder } from './custom-http-param-encoder'; // Correct path
+
+/**
+ * Builds HttpParams from a nested object or array recursively.
+ */
+export function buildHttpParams(query: any, prefix: string = ''): HttpParams {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)